### ISSUE TITLE
Question about  CIN  part

### ISSUE THREAD

**Issue by Lukelluke** [SamuelBroughton/StarGAN-Voice-Conversion-2#9]
Hi, @SamuelBroughton ,nice to issue u again

1. I recently noticed that, in the CIN part, ur implements is different to original vc2 paper, where mentioned that the `$\ beta_{c'}$` and `$\gamma_{c'}$` are replaced as`$\beta_{c,c'}$` and `$\gamma_{c,c'}$ `

2. And I noticed that in some other image field implements, such as [this](https://github.com/ChinmayLad/neural-style-transfer/blob/dc44cd261fd9bdf684440ef104cec65fe5be15c5/normalization.py),  or [tensorflow version](https://github.com/MingtaoGuo/Conditional-Instance-Norm-for-n-Style-Transfer/blob/master/ops.py#L5) implement the CIN part different as our thoughts. At least we should use 
`F.instance_norm(
            input, self.running_mean, self.running_var, w = gamma_t, b = beta_t,
            self.training or not self.track_running_stats,
            self.momentum, self.eps)`
And in original paper, it mentioned that `and the momentum term was set to 0.5` at  Part 4.1 in Page 4, which i think shoudl be use in all Instant Norm layers, both CIN and IN.

3. BTW, the original paper mentioned that, `These parameters are learnable and optimized through training. ` , and we use the `nn.linear()` function, which can change  w && b   into `parameter` which is trainable, the same as `nn.Parameter()`, does my description  right ?

Hope to get ur reply again, my friend

All the best,
Luke Huang

**Comment by shackysureshot** [comment]
1. See my comment at the top of the README: "Uses source and target domain codes in D but not G as I found better quality output"
2. All implementations won't be identical. I believe the authors are referring to the momentum term used for SGD
3. Yes, parameters get added to the module attributes
